lapis/job.py

```diff
@@ -87,12 +92,25 @@ def waiting_time(self) -> float:
             return self.in_queue_until - self.in_queue_since
         return float("Inf")
 
+    def modified_walltime(self):
+        if self.fileprovider and self.fileprovider.provides_all_files(self):
+            return walltime_models["maxeff"](self)
+        else:
+            return self.walltime
+
     async def run(self):
         self.in_queue_until = time.now
         self._success = None
         await sampling_required.put(self)
+        if self.drone:
+            print(
+                "running job {} on site {} in drone {}".format(
+                    repr(self), self.drone.sitename, repr(self.drone)
+                )
+            )
```

Comment:
Did it ever happen to you that a job was running without being assigned to a proper drone? We should consider this an error if it did :)

Reply:
This happens in the unit tests... Same with the fileprovider in 
https://github.com/tfesenbecker/lapis/blob/53ebec5ea004e4f031a0f70102276fe0102d62e4/lapis/job.py#L96

Reply:
Ok, thanks. I will create an issue for this.